README.md

```diff
@@ -75,7 +90,9 @@ and `prev_article_in_subcategory#` where `#` is the level of subcategory. So
 using the example from above, `subcategory1` will be `Foo`, and `subcategory2`
 will be `Foo/Bar`.
 
-Therefor the usage with subcategories is:
+## template examples
+
+The usage with subcategories from either `subcategory` or `more_categories` is:
```

Comment:
from either `subcategory` or `more_categories` is → from either the Subcategory plugin or the More Categories plugin is: